src/sentry/integrations/vercel/integration.py

```diff
@@ -203,7 +204,11 @@ def update_organization_config(self, data):
         # data = {"project_mappings": [[sentry_project_id, vercel_project_id]]}
         vercel_client = self.get_client()
         config = self.org_integration.config
-        new_mappings = data["project_mappings"]
+        try:
+            new_mappings = data["project_mappings"]
+        except KeyError:
+            return HttpResponse("Failed to update configuration.")
```

Comment:
@ceorourke we should return a 400 error. I think that a `ValidationError` would be appropriate here.